safe/impact_functions/volcanic/volcano_polygon_building/impact_function.py

```diff
@@ -46,6 +46,9 @@ def __init__(self):
         self.volcano_names = tr('Not specified in data')
         self._target_field = 'Hazard'
 
+        # From BuildingExposureReportMixin
+        self.building_report_threshold = 25
```

Comment:
After some thought, perhaps we can put the initialisation in `building_exposure_report_mixin.py`.